src/pytest_regressions/common.py

```diff
@@ -69,42 +80,42 @@ def check_text_files(obtained_fn, expected_fn, fix_callback=lambda x: x, encodin
 
 
 def perform_regression_check(
-    datadir,
-    original_datadir,
-    request,
-    check_fn,
-    dump_fn,
-    extension,
-    basename=None,
-    fullpath=None,
-    force_regen=False,
-    with_test_class_names=False,
-    obtained_filename=None,
-    dump_aux_fn=lambda filename: [],
-):
+    datadir: Path,
```

Comment:
I don't remember, did we implicitly accept `Path | str` for these parameters before (undocumented)? If so, this could be a breaking change, I'm not sure if we should maintain a "flexible" API...

Reply:
This module `common` is only used internally by the fixtures themselves, so we are fine.